src/local-peers.js

```diff
@@ -27,6 +28,7 @@ const DEDUPE_TIMEOUT = 1000
 const MESSAGE_TYPES = {
   Invite: 0,
   InviteResponse: 1,
+  ProjectJoinDetails: 3,
```

Comment:
Based on the comment above, I didn't reorder this and just added this as message type 3. Let me know if I should reorder given that we haven't released.

Reply:
not sure which comment you are referring to? I think for something like this maybe either re-number to `2` if not released, or else keep the removed type in the list with a comment explaining it's deprecated, otherwise someone in the future looks at this and spends a while scratching their head about how it ended up like this.

Reply:
Reordered in 34465832688eb641bb8c37483b85469d2b244a84 (leaving a space for invite cancellations).